src/utils/components/NetworkInterfaceModal/utils/helpers.ts

```diff
@@ -63,3 +66,13 @@ export const createInterface = (
     name: nicName,
   };
 };
+
+export const getNadType = (nad: NetworkAttachmentDefinition): string => {
+  try {
+    const config = JSON.parse(nad?.spec?.config);
+    //can be config.type or config.plugin first element only!'
+    return interfacesTypes?.[config?.type] || interfacesTypes?.[config?.plugins?.[0]?.type];
+  } catch (e) {
+    console.log('Cannot convert NAD config: ', e);
```

Comment:
do we want this as console.log?

Reply:
yes if something failed why parsing from BE i want to know